axelrod/fingerprint.py

```diff
@@ -0,0 +1,294 @@
+import axelrod as axl
+import numpy as np
+import matplotlib.pyplot as plt
+from axelrod.strategy_transformers import JossAnnTransformer, DualTransformer
+from axelrod.interaction_utils import compute_final_score_per_turn as cfspt
```

Comment:
Could we leave this as `compute_final_score_per_turn`, I had no idea what `cfspt` was when reading lower down.